### DISCUSSION TITLE
Disabling realtime (Phoenix)

### DISCUSSION THREAD

**Discussion by ManUtopiK** [supabase/supabase#2496] [Questions]
Is there a way to disable real time functionality? I don't need it.
If disabled, can I still use the supabase UI?

**Comment by silentworks** [comment]
Realtime is disabled by default from what I can remember. Yes you can still use the Supabase UI even if realtime is disabled.

It's also not clear if you are talking about realtime in the supabase-{js,python,etc} libraries or if you are talking about the realtime updates in the Supabase UI. Can you clarify this if my answer above didn't answer the questions asked.

**Reply by ManUtopiK** [reply]
I would like to disable Elixir server (Phoenix) in Kong to not supercharge the server.
Thanks for your response!

**Reply by silentworks** [reply]
It's still not very clear what you are asking here, are you making reference to the self hosted version of supabase or the hosted version? As stated before in the hosted version realtime is not enabled by default when using the supabase libraries.

If you are asking about disabling it for the Supabase Dashboard UI, then I don't think this is possible at the moment and it shouldn't have any effect on you using the service either.
